src/tests/ftest/pool/dynamic_server_pool.py

```diff
@@ -100,7 +99,7 @@ def test_dynamic_server_pool(self):
 
         :avocado: tags=all,full_regression
         :avocado: tags=vm
-        :avocado: tags=control,dynamic_server_pool
+        :avocado: tags=pool,control,dynamic_server_pool
```

Comment:
This is okay, but we prefer to put "features" on a separate line
```suggestion
        :avocado: tags=pool,control
        :avocado: tags=dynamic_server_pool
```

Reply:
Done, was wondering about that.

Reply:
Thanks!